src/Activity.js

```diff
@@ -1,19 +1,22 @@
+import Utility from "./Utility";
+
 class Activity extends Utility {
-  constructor(dataSet, userID) {
+  constructor(dataSet, userID, userData) {
     super (dataSet, userID);
+    this.userData = userData
   }
 
   returnMilesWalked() {
-    return Number((this.user.strideLength * this.singleUserData[this.singleUserData.length - 1].numSteps / 5280).toFixed(2))
+    return Number((this.userData.strideLength * this.singleUserData[this.singleUserData.length - 1].numSteps / 5280).toFixed(2))
```

Comment:
I just want to make sure I understand this change.   The object being passed through as a parameter is the same user object that they were originally passing through correct?   They called it `user` and we've just renamed it to `userData`.  Is that correct?

Before and after, this is the object being passed through because we need to be able to get the goal/stridelength/friends that live in this user object?

```
{ "id": 1,
   "name": "Luisa Hane",
   "address": "15195 Nakia Tunnel, Erdmanport VA 19901-1697",
   "email": "Diana.Hayes1@hotmail.com",
   "strideLength": 4.3,
   "dailyStepGoal": 10000,
   "friends": [ 2, 3, 4 ] }
```

Reply:
You've absolutely got it. Sorry for the delayed response.
Closing out this thread.
-AD